README.md

```diff
@@ -190,95 +184,85 @@ package my-org:my-component
 /// An example world for the component to target.
 world example {
     export hello-world: func() -> string
-}                
+}
 ```
 
 The component will export a `hello-world` function returning a string.
 
 The implementation of the component will be in `src/lib.rs`:
 
 ```rust
+cargo_component_bindings::generate!();
+
+use bindings::Example;
+
 struct Component;
 
-impl bindings::Example for Component {
+impl Example for Component {
     /// Say hello!
     fn hello_world() -> String {
         "Hello, World!".to_string()
     }
 }
-
-bindings::export!(Component);
 ```
 
-Here `bindings` is the bindings crate that `cargo component` generated for you.
+The `generate!` macro is responsible for generating the bindings to allow the
+Rust code to export what is expected of the component.
 
-The `export!` macro informs the bindings that the `Component` type exports
-all interfaces listed in `Cargo.toml`.
+It generates a Rust module named `bindings` containing the types and traits the
+correspond to the world definition.
```

Comment:
One slightly subtle consequence of the change from a bindings crate to a bindings module is that the generated module is not accessible outside the module that invokes the macro. That is, you get a `mod`, and sometimes a `pub(crate) mod` would be useful. I'm not sure offhand what to suggest for that though.

Reply:
I think a `pub(crate)` on the bindings module would make sense and if they need to be public for some reason, the module containing the `generate!` macro can reexport them.